app/components/form_status_tag_component/view.rb

```diff
@@ -1,17 +1,21 @@
 module FormStatusTagComponent
   class View < ViewComponent::Base
-    attr_accessor :status
-
-    def initialize(status: "DRAFT")
+    def initialize(status: :draft)
       super
-      @status = status.upcase
+      @status = status.to_sym
```

Comment:
Could we add some tests to show that we convert the status to a symbol? Just wondering what would happened if `to_sym` was accidentally removed and some call to this component passed in a string somewhere

Reply:
Added